lib/utils/session_vars.js

```diff
@@ -21,9 +21,15 @@ const session_vars = {};
 module.exports = session_vars;
 
 // These default to null
+// TODO: How to isolate debug elsewhere so this file can use the logger?
 session_vars.get_debug = function () { return process.env.DEBUG || null; };
 session_vars.get_dev_api_key = function () { return process.env.DOCASSEMBLE_DEVELOPER_API_KEY || null; };
 session_vars.get_repo_url = function () { return process.env.REPO_URL || null; };
+// .is_local more appropriate? To manage storing local results differently considering
```

Comment:
IMO .is_dev_env is fine. It describes the why (because we're re-running scenarios without setup each time), and not the how (locally, which could technically be through an SSH'd server and not locally).

Reply:
```suggestion
```